src/context/operationsProvider.tsx

```diff
@@ -40,7 +43,9 @@ const OperationsProvider: FC<Props> = ({ children }) => {
   } = useQuery({
     queryKey: [queryKeys.operations],
     queryFn: () => fetchAllOperations(),
+    enabled: isAuthenticated,
```

Comment:
thanks for implementing the auth check here as well as in `StatusBar`!

Reply:
We missed the login cases, when introducing those. Should keep those in mind next time we do something global.